apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -26,11 +26,11 @@ import {
   SigningServiceResponse,
 } from '@island.is/dokobit-signing'
 import { InjectQueue, QueueService } from '@island.is/message-queue'
+import { MessageType } from '@island.is/judicial-system/message'
```

Comment:
`MessageType` and `completedCaseStates` are not being used in this file. Please remove them.

Reply:
They are not used because we commented out the queuing functionality. I will comment these out as well, but not remove them.